src/components/Table/TableFormatters/SecretViewerFormatter.vue

```diff
@@ -171,6 +171,7 @@ export default {
       }
     },
     onEditBlur() {
+      this.isEdit = false
       this.$emit('input', this.realValue)
     }
   }
```

Comment:
The given code seems to be JavaScript code from a Vue component, possibly related to handling input value changes.

Here's an overview of some key differences compared with previous versions:

- **Variables**
  - The variables `realValue`, `value`, and maybe others have slightly different names in this version, which could lead to confusion or errors if not kept consistent.

  *In previous versions, `realValue` is used.*

  ```javascript
function Input(
```